### PR TITLE
feat: upload queue on mobile

### PR DESCRIPTION
**Pull request by y-lohse**
Hi @GoOz! I think I sort of stole this card from you, but I felt like a css challenge. #sorrynotsorry

It's mostly a big css reset since the natural behavior is close to what we're expecting on mobile.

### CODE REVIEW

**Comment by y-lohse** [comment]
[(2) [PHO] Améliorer l'intégration de l'upload en responsive](https://trello.com/c/thFup14g/338-2-pho-am%C3%A9liorer-lint%C3%A9gration-de-lupload-en-responsive)

File: src/ducks/upload/styles.styl [hunk 1]
```diff
@@ -135,3 +135,40 @@ progress.upload-queue-progress::-moz-progress-bar
...
+        background 0
+        box-shadow none
+
+        &:before
+            position fixed
+            top $coz-bar-size - $coz-bar-border-size
+            z-index  $nav-index
+            width 100%
+            height $coz-bar-border-size
+            content ""
```
Comment (GoOz) [inline src/ducks/upload/styles.styl]: Please use single quotes. :)

File: src/ducks/upload/styles.styl [hunk 2]
```diff
@@ -135,3 +135,40 @@ progress.upload-queue-progress::-moz-progress-bar
...
+            background white
+
+    .upload-queue--visible
+        max-height 30%
```
Comment (GoOz) [inline src/ducks/upload/styles.styl]: I just don't get this one, what is is for exactly?

Comment (y-lohse) [inline src/ducks/upload/styles.styl]: `.upload-queue` is never removed from the DOM; on desktop, it's always there but outside the viewport, and it transitions in when an upload starts. So on mobile, I used a "slide down" sort of animation to make it appear (it has a `max-height` of 0 by default). `30%` is probably way too much now, initially I was also displaying the details of the uploaded files, but that's no longer the case.
Do you see another approach or a better animation for this?

Comment (GoOz) [inline src/ducks/upload/styles.styl]: Oh ok yeah i get it now :) No problem, that's what I usually do too.

**Pull request merged**